planetiler-core/src/main/java/com/onthegomap/planetiler/collection/FeatureGroup.java

```diff
@@ -60,27 +61,55 @@ public final class FeatureGroup implements Iterable<FeatureGroup.TileFeatures>,
   private final Stats stats;
   private final LayerStats layerStats = new LayerStats();
   private volatile boolean prepared = false;
+  private final TileOrder tileOrder;
 
-  FeatureGroup(FeatureSort sorter, Profile profile, Stats stats) {
+  public enum TileOrder {
```

Comment:
What do you think about moving TileOrder into/alongside TileCoord? It seems closely bound to that even though it primarily gets used here.

Reply:
moved to `geo.TileOrder` file